src/test/java/doublelinkedlist/DoubleLinkedListTest.java

```diff
@@ -12,7 +12,8 @@ public class DoubleLinkedListTest {
 
     @BeforeEach
     void setUp() {
-        doubleLinkedList = new DoubleLinkedList(1);
+        Node node = new Node(1);
+        doubleLinkedList = new DoubleLinkedList(node);
```

Comment:
You can redundant your line code with shorten implement. You don't reuse node variable anywhere, so no need to definition it.
     doubleLinkedList = new DoubleLinkedList(new Node(1));